sqlite/comdb2build.c

```diff
@@ -2311,6 +2312,13 @@ static char *gen_key_name(struct comdb2_ddl_context *ctx, const char *tabname,
     return keyname;
 }
 
+#define PK "PRIMARY_KEY"
```

Comment:
Consider using `COMDB2_PK` or similar to avoid name clashes with existing instances.